src/module/comment_response_messages.js

```diff
@@ -0,0 +1,30 @@
+export const commentAddSuccess = () => {
+  document.querySelectorAll('.input').forEach((input) => {
+    input.value = '';
+  });
+  const response = document.querySelector('.response-message');
+  response.classList.remove('hide');
+  response.classList.remove('red');
+  response.classList.add('green');
+  response.innerHTML = 'Comment added successfully!!!';
+  setTimeout(() => {
+    response.classList.add('hide');
+    response.classList.remove('green');
+    response.classList.remove('red');
+    response.innerHTML = '';
+  }, 5000);
+};
+
+export const commentAddError = () => {
+  const response = document.querySelector('.response-message');
+  response.classList.remove('hide');
+  response.classList.remove('green');
```

Comment:
- [ ]  **Please Scan your code and remove all the unnecessary lines**

Reply:
Ow! I overlooked this earlier :smile:  but it's now fixed!
Thanks!!